theming/_variables.scss

```
@@ -1,38 +1,210 @@
+  min-height: 100vh;
+  flex-direction: column;
+}
+
+.center-image {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+//// Homepage //////
+.gradient-background {
+  background: linear-gradient(
+    180deg,
+    rgba(0, 112, 192, 0.07) 58.33%,
+    rgba(255, 255, 255, 0) 100%
+  );
+}
+
+.welcome-text {
+  font-weight: 500;
+  font-size: 48px;
+  line-height: 56px;
+  text-align: left;
+}
+
+.homepage-tile {
+  background: #ffffff;
+  border-radius: 22px;
+  filter: drop-shadow(-3px 11px 44px rgba(0, 0, 0, 0.2));
+}
+
+.homepage-tile:hover {
+  filter: drop-shadow(-3px 11px 44px rgba(0, 0, 0, 0.3));
+}
+
+.center {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+.intro {
+  margin-right: 320px !important;
+}
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  [class*="intro"] {
+    margin-right: 0 !important;
+  }
+}
+
+.homepage-tile {
+  width: 100%;
+  min-width: 300px;
+  overflow-x: hidden;
+}
+
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  [class*="homepage-tile"] {
+    margin-bottom: 150px !important;
+  }
+}
+
+.benefits-text {
+  color: #0070c0;
+
+  font-weight: 500;
+  font-size: calc(4px + 1.5vw);
+}
+
+.temoc-level {
+  margin-bottom: 0 !important;
+  margin-left: 4%;
+  margin-right: 4%;
+
+  margin-bottom: 2vw !important;
+}
+
+@media only screen and (max-width: 1400px) {
+  /* For mobile phones: */
+  [class*="temoc-level"] {
+    margin-bottom: 40px !important;
+    margin-top: 50px !important;
+  }
+}
+
+.benefits-p {
+  color: rgba(0, 0, 0, 0.54);
+
+  padding-top: 3vw;
+  padding-left: 10vw;
+  padding-right: 10vw;
+}
+
+.small-res-newsletter-form{
+  display: none;
+  margin-bottom: 35px !important;
+}
+
+@media only screen and (max-width: 1400px) {
+  /* For mobile phones: */
+  [class*="small-res-newsletter"] {
+    display: initial;
+  }
+}
+
+.big-res-newsletter-form{
+  display: initial;
+}
+
+@media only screen and (max-width: 1400px) {
+  /* For mobile phones: */
+  [class*="big-res-newsletter-form"] {
+    display: none !important;
+  }
 }
+/////////////////////////////////////////////////////
 
-.center-image{
-    margin-left: auto;
-    margin-right: auto;
+//////////////////Footer//////////////////////
+.footer-text {
+  color: rgba(0, 0, 0, 0.6);
+  margin-bottom: 16px;
+  // text-align: left;
+  // text-indent: inherit;
+  // display: inline-block;
 }
 
+.footer-title {
+  color: $primary;
+  margin-bottom: 24px !important;
+  // text-align: justify;
+  // float: center;
+}
+
+a {
+  color: rgba(0, 0, 0, 0.6) !important;
+  background-color: transparent;
+  text-decoration: none;
+}
+
+.footer-link:hover {
+  color: $primary !important;
+  background-color: transparent;
+  text-decoration: underline;
+}
+
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  [class|="footer"] {
+    text-align: center !important;
+  }
+}
```

I think it would also probably best to put all the footer classes next to each other to make it clearer which classes it applies to (pretty much swap the `a { ... }` with `.footer-link:hover { .... }` and then I'd probably put this one right above the first footer class
```suggestion
/* Applies to all footer[-something] classes. ex: footer-title, footer-link, etc. */
[class|="footer"] {
  /* For mobile phones: */
  @media only screen and (max-width: 768px) { 
    text-align: center !important;
  }
}
```